core/App/screens/CredentialDetails.tsx

```diff
@@ -107,25 +108,31 @@ const CredentialDetails: React.FC<CredentialDetailsProps> = ({ navigation, route
     if (!indyCredentialFormat) {
       return
     }
-    const isRevoked = revoked.has(indyCredentialFormat.credentialRecordId)
-    setIsRevoked(isRevoked)
-    const isRevokedMessageDismissed = revokedMessageDismissed.has(indyCredentialFormat.credentialRecordId)
-    setIsRevokedMessageHidden(isRevokedMessageDismissed)
+    credential.revocationNotification == undefined ? setIsRevoked(false) : setIsRevoked(true)
+    if (isRevoked && credential?.revocationNotification?.revocationDate) {
+      const date = new Date(credential.revocationNotification.revocationDate)
+      setRevocationDate(date.toLocaleDateString('en-CA', dateFormatOptions))
+    }
     OCABundle.getCredentialPresentationFields(credential as CredentialExchangeRecord, getCurrentLanguage()).then(
       (fields) => setFields(fields)
     )
   }, [credential])
 
+  useEffect(() => {
+    if (credential) {
```

Comment:
Small nitpick, but could we just remove the outer conditional check and just use `credential?.revocationNotification`?

Reply:
Thanks for catching these, they should both now be updated